arangod/Agency/State.cpp

```diff
@@ -1864,13 +1894,15 @@ uint64_t State::toVelocyPack(index_t lastIndex, VPackBuilder& builder) const {
 /// @brief dump the entire in-memory state to velocypack.
 /// should be used for testing only
 void State::toVelocyPack(velocypack::Builder& builder) const {
-  std::lock_guard mutexLocker{_logLock};
-
   builder.openObject();
   builder.add("current", VPackValue(_cur));
```

Comment:
Is it safe to read `_cur` without lock?

Reply:
No, it's not safe. I overlooked the access to `_cur` here. Good catch!